core/src/main/scala/kafka/log/LogConfig.scala

```diff
@@ -19,6 +19,7 @@ package kafka.log
 
 import java.util.Properties
 import kafka.server.KafkaConfig
+import org.apache.kafka.common.errors.InvalidEntityConfigurationException
 import org.apache.kafka.common.utils.Utils
 import scala.collection._
 import org.apache.kafka.common.config.{AbstractConfig, ConfigDef}
```

Comment:
I only added one import line here. It is next to another o.a.k import and sorted alphabetically. Should I be refactoring the all imports in any touched file?